.travis.yml

```diff
@@ -3,11 +3,13 @@ matrix:
   include:
     - os: osx
       osx_image: xcode9      
-      sudo: required      
-      dotnet: 2.0.0    
+      sudo: required        
 before_install:
   - brew update
+  - curl https://dot.net/v1/dotnet-install.sh -o dotnet-install.sh
+  - chmod +x dotnet-install.sh
+  - ./dotnet-install.sh --channel 2.0
 install:
   - ulimit -n4096  
 script:  
-  - dotnet test src/Dotnet.Script.Tests/Dotnet.Script.Tests.csproj
\ No newline at end of file
+  - ~/.dotnet/dotnet test src/Dotnet.Script.Tests/Dotnet.Script.Tests.csproj
```

Comment:
should we now run here `build.sh` instead?

Reply:
Yeah, that was just a temporary thing. I ran into issues with needing the 1.1 sdk for running dotnet.script and the 2.0 sdk for compiling :) This is no longer an issue since we have releases 0.14.0 which is netcore2.0. I'll fix